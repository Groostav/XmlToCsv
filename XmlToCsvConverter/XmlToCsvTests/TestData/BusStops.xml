<BusStops>
	<BusStop>
		<Number>15</Number>
		<Name>Hastings</Name>
	</BusStop>
  <BusStop>
		<Number>16</Number>
		<Name>Arbutus</Name>
	</BusStop>
</BusStops>