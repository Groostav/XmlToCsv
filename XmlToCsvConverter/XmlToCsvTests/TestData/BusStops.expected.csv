Number,Name
"15","Hastings"
"16","Arbutus"
